### PR TITLE
Aj/description optional for publish workflow

### PR DESCRIPTION
**Pull request by astuyve**
- Optionally add a description to the layer, useful for tracking branch names for RCs, agent version numbers, and potentially useful for production builds
- Passes ownership of args to commands, as they are not reused by `main`.

### CODE REVIEW

File: .github/workflows/deploy-sandbox.yml [hunk 1]
```diff
@@ -122,7 +126,8 @@ jobs:
           --layer-name "Datadog-Extension" \
           --layer-suffix "${{ inputs.layerSuffix }}" \
           --region ${{ github.event.inputs.region }} \
-          --key ${{ secrets.GH_ACTION_PUBLISHER_AEM_KEY }}
+          --key ${{ secrets.GH_ACTION_PUBLISHER_AEM_KEY }} \
+          --description "${{ inputs.description }}"
```
Comment (maxday) [inline .github/workflows/deploy-sandbox.yml:130]: did you the GitHub action when description is empty?

Comment (astuyve) [reply .github/workflows/deploy-sandbox.yml:130]: I don't know how to test this actually! is there a good way?

Comment (maxday) [reply .github/workflows/deploy-sandbox.yml:130]: you can write a integration test or manually trigger the workflow locally using https://cli.github.com/manual/gh_workflow_run

**Comment by duncanista** [comment]
Closing due to #295 and tools have been moved to [DataDog/serverless-tools](https://github.com/DataDog/serverless-tools).

**Pull request closed without merging**